src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -490,6 +490,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 			showHealthCertificate: { [weak self] route in
 				// We must NOT call self?.showHome(route) here because we do not target the home screen. Only set the route. The rest is done automatically by the startup process of the app.
 				self?.route = route
+			}, showHealthCertifiedPerson: { [weak self] route in
+				// We must NOT call self?.showHome(route) here because we do not target the home screen. Only set the route. The rest is done automatically by the startup process of the app.
+				self?.showHome(route)
```

Comment:
this can lead to unexpected behavior. Did you try to just use `self.rout = route`

Reply:
yes, one of the cases where we trigger the notification is when we enter foreground, that way we are already inside the app "not on app start"

Reply:
The comment does not match the implementation.

Reply:
I corrected the comment to describe the case, thanks